back/engines/commercial/content_builder/spec/acceptance/layouts_spec.rb

```diff
@@ -0,0 +1,406 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'rspec_api_documentation/dsl'
+
+resource 'ContentBuilderLayouts' do
```

Comment:
This is really great test coverage! But currently all these examples will be documented in the API docs, which I think is a bit too much. To leave out examples from the API docs, you can change `example_request` to `example` and specify `document: false`.

Reply:
> This is really great test coverage!

Thank you 😃  I tried to cover all aspects.

Why do you think it is a bit too much? What aspects of the API would you exclude from the documentation? Isn't it better to document all aspects of an API, not only the happy paths, but also what the response is when something goes wrong? Let's talk about it.